src/Accounting/index.ts

```diff
@@ -58,16 +61,12 @@ export const calcProfitableLiquidationPrice: (
     price: number,
     maxLeverage: number,
 ) => number = (quote, base, price, maxLeverage) => {
-    const margin = totalMargin(base, quote, price);
     const borrowed = calcBorrowed(quote, base, price);
-    if (borrowed > 0 || margin < 0) {
-        return quote > 0 // case 1
-            ? (maxLeverage * (base - (RYAN_6 * LIQUIDATION_GAS_COST - LIQUIDATION_GAS_COST))) /
-                  (quote - maxLeverage * quote)
-            : 0 + quote < 0 // case 2
-            ? -1 *
-              ((base * (maxLeverage - (RYAN_6 * LIQUIDATION_GAS_COST - LIQUIDATION_GAS_COST) * maxLeverage)) /
-                  (maxLeverage * quote + quote))
+    if (borrowed > 0 || base < 0) {
+        return base > 0 // case 1
+            ? (maxLeverage * (quote - (RYAN_6 * LIQUIDATION_GAS_COST - LIQUIDATION_GAS_COST))) / (base - maxLeverage * base)
+            : 0 + base < 0 // case 2
+            ? (-1 * (quote * maxLeverage - (RYAN_6 * LIQUIDATION_GAS_COST - LIQUIDATION_GAS_COST) * maxLeverage)) / (maxLeverage * base + base)
```

Comment:
Can you define these in variables, so we know what they are supposed to represent please :100:

Reply:
which ones the 0's and the -1??

Reply:
The whole things. Like what does the numerator represent, and the denominator? I get that these are the formulas for calculating the liquidation price, but anybody who hasn't been able to go through the accounting doc is going to have a really hard time figuring out how this works, nothing is named